site/docs/stylesheets/extra.css

```
@@ -21,3 +21,31 @@ img.bordered {
 .md-typeset td code {
   word-break: revert;
 }
+
```

Mind adding this as well?

```suggestion

.md-typeset .tabbed-set>.tabbed-content {
  padding: 0 .5em 0 .5em;
  border: 1px solid #ababab;
  border-bottom-left-radius: .3em;
  border-bottom-right-radius: .3em;
}

```

---

Quite frankly I don't like it :-D but I added it nonetheless.